src/main/java/uk/gov/hmcts/ccd/domain/service/processor/DisplayContextParameterType.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.ccd.domain.service.processor;
+
+import com.google.common.base.Strings;
+
+import java.util.*;
+import java.util.regex.Matcher;
```

Comment:
you can organise imports in inteliJ and add the option to  implicit add each import . It is very cosmetic but sometimes checkstyle tends to complaint about " .* " .